roles/kubespray-defaults/defaults/main/download.yml

```diff
@@ -1031,6 +1031,14 @@ downloads:
     groups:
       - kube_node
 
+  ingress_nginx_kube_webhook_certgen:
+    repo: "{{ ingress_nginx_kube_webhook_certgen_image_repo }}"
+    tag: "{{ ingress_nginx_kube_webhook_certgen_image_tag }}"
+    sha256: "{{ ingress_nginx_kube_webhook_certgen_digest_checksum | default(None) }}"
+    groups:
+      - kube_node
+    when: ingress_nginx_webhook_enabled
```

Comment:
I think this should be `enabled: "{{ ingress_nginx_webhook_enabled }}"` like the other in this file.

Reply:
Thank you for comments!
I fixed it as descibed below.
```
enabled: "{{ ingress_nginx_webhook_enabled }}"
```
Dose this address your cocmments?
commit: d47e05147a3a542325d7b3e9ceaaca083a20ed10